core/src/main/java/com/linecorp/armeria/client/ClientFactoryBuilder.java

```diff
@@ -427,6 +427,16 @@ public ClientFactoryBuilder idleTimeoutMillis(long idleTimeoutMillis) {
         return this;
     }
 
+    /**
+     * Sets the max age of a socket connection in milliseconds. The connection is closed
+     * after this amount of time of creation.
+     */
+    public ClientFactoryBuilder maxAgeMillis(long maxAgeMillis) {
```

Comment:
- `maxConnectionAgeMillis()`?
- Could you also add `maxConnectionAge(Duration)`?